builtin/locale/__builtin.it.tr

```diff
@@ -140,121 +140,107 @@ This command was disabled by a mod or game.=Questo comando è stato disabilitato
 Invalid usage, see /help clearobjects.=Uso incorretto, vedi /help clearobjects.
-Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Eliminando tutti gli oggetti/entità. Questo potrebbe richiedere molto tempo e farti eventualmente crashare. (di @1)
+Clearing all objects. This may take a long time. You may experience a timeout. (by @1)=Eliminando tutti gli oggetti/entità. Questo potrebbe richiedere molto tempo e farti disconnettere. (di @1)
 Cleared all objects.=Tutti gli oggetti sono stati eliminati.
 <name> <message>=<nome> <messaggio>
-Send a direct message to a player=Invia un messaggio privato al giocatore
+Send a direct message to a player=Invia un messaggio privato a unə giocatore
 Invalid usage, see /help msg.=Uso incorretto, vedi /help msg
-The player @1 is not online.=Il giocatore @1 non è connesso.
+The player @1 is not online.=Lə giocatore @1 non è connessə.
 DM from @1: @2=Messaggio privato da @1: @2
 Message sent.=Messaggio inviato.
-Get the last login time of a player or yourself=Ritorna l'ultimo accesso di un giocatore o di te stessǝ
-@1's last login time was @2.=L'ultimo accesso di @1 è avvenuto il @2
-@1's last login time is unknown.=L'ultimo accesso di @1 non è conosciuto
-Clear the inventory of yourself or another player=Svuota l'inventario tuo o di un altro giocatore
-You don't have permission to clear another player's inventory (missing privilege: @1).=Non hai il permesso di svuotare l'inventario di un altro giocatore (privilegio mancante: @1).
+Get the last login time of a player or yourself=Ritorna l'ultimo accesso di unə giocatore o di te stessǝ
+@1's last login time was @2.=L'ultimo accesso di @1 è avvenuto il @2.
+@1's last login time is unknown.=L'ultimo accesso di @1 è ignoto.
+Clear the inventory of yourself or another player=Svuota l'inventario tuo o di un*altrə giocatore
+You don't have permission to clear another player's inventory (missing privilege: @1).=Non hai il permesso di svuotare l'inventario di un*altrə giocatore (privilegio mancante: @1).
 @1 cleared your inventory.=@1 ha svuotato il tuo inventario.
 Cleared @1's inventory.=L'inventario di @1 è stato svuotato.
-Player must be online to clear inventory!=Il giocatore deve essere connesso per svuotarne l'inventario!
-Players can't be killed, damage has been disabled.=I giocatori non possono essere uccisi, il danno è disabilitato.
-Player @1 is not online.=Il giocatore @1 non è connesso.
+Player must be online to clear inventory!=Lə giocatore deve essere connessə per svuotarne l'inventario!
+Players can't be killed, damage has been disabled.=Lɜ giocatori non possono essere uccisɜ, il danno è disabilitato.
+Player @1 is not online.=Lə giocatore @1 non è connesso.
 You are already dead.=Sei già mortǝ.
 @1 is already dead.=@1 è già mortǝ.
 @1 has been killed.=@1 è stato uccisǝ.
-Kill player or yourself=Uccide un giocatore o te stessǝ
+Kill player or yourself=Uccide unə giocatore o te stessǝ
 Invalid parameters (see /help @1).=
```

Comment:
This is missing a translation.

Reply:
Done. the `*` is the proposed inclusive solution in https://italianoinclusivo.it/. In Italian, the indefinite article `un` can go both with feminine and masculine words if they start with a vowel. However, only feminine names take an apostrophe after `un`, so "un amico (a male friend)" but "un'amica (a female friend). The inclusive proposal is to use an asterisk (so a different symbol) to avoid creating a mental inference in the reader: "un*amicə"

Reply:
Ah, I see. That also explains the unusual `ǝ` and `ɜ` characters.